package-manifest.toml

```diff
@@ -248,10 +250,10 @@ only_for_targets.image = "standard"
 source.type = "local"
 source.rust.binary_names = ["clickhouse-admin-keeper", "clickhouse-admin-server"]
 source.rust.release = true
-source.paths = [
-  { from = "smf/clickhouse-admin-keeper", to = "/var/svc/manifest/site/clickhouse-admin-keeper" },
-  { from = "smf/clickhouse-admin-server", to = "/var/svc/manifest/site/clickhouse-admin-server" },
-]
+# We specifically put the smf manifests with their corresponding container zones
+# so that both servers aren't started simultaneously
+# See [package.clickhouse_keeper_svc] and [package.clickhouse_server_svc]
+source.paths = []
```

Comment:
Does this mean we're installing two clickhouse-admin binaries in each zone? Perhaps we may want to separate this package into two and only install a single binary in each zone with the corresponding name? WDYT?

Reply:
We could, but it seems like more tedium. `omicron-pacakge` is the only thing that cares, and this seems to work fine.